### PR TITLE
refactor: Background Image. Eager load for top image for new pages

### PR DESCRIPTION
**Pull request by yashim-deriv**
Changes:
-  Refactored background image component to typescript. updated loading attribute for new pages

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [x] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_signup-simple.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ const SignupSimple = ({
             <Container>
                 <StyledHeader weight="bold">
                     {localize('Ready to join us?')}
-                    <br></br>
+                    <br />
```
Comment (habib-deriv) [inline src/components/custom/_signup-simple.js:57]: Can we add a new tag and apply css instead on br?

File: src/components/elements/query-image.tsx [hunk 2]
```diff
@@ -64,6 +64,7 @@ const QueryImage = ({
                 disable_transition={disable_transition}
                 onClick={onClick}
             >
+                <div style={{ width: '90px' }}></div>
```
Comment (sean-binary) [inline src/components/elements/query-image.tsx:67]: 👀

File: src/pages/who-we-are/components/_hero.tsx [hunk 3]
```diff
@@ -88,16 +87,12 @@ const StyledMobileQueryImage = styled(QueryImage)`
...
-        ) : (
-            <>
-                {localize('Who')}
-                <br></br>
-                {localize('we are')}
-            </>
-        )
+    const title = (
+        <Flex tabletS={{ fd: 'column' }}>
+            <Box mr="3.8rem">{localize('Who')}</Box>
```
Comment (habib-deriv) [inline src/pages/who-we-are/components/_hero.tsx:92]: This 2 needs to be in same div, for localization

Comment (yashim-deriv) [reply src/pages/who-we-are/components/_hero.tsx:92]: Apparently it was probably a requirement 😅  should we maintain without breaks? @habib-deriv

Comment (yashim-deriv) [reply src/pages/who-we-are/components/_hero.tsx]: Reverted due to localization 🙏🏼

**Review by habib-deriv — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]
LGTM

**Pull request merged**